From a5cd2721167fc32d497582bc87cc43f4da99caa7 Mon Sep 17 00:00:00 2001
From: sorinrevsw <sorin@revsw.com>
Date: Fri, 23 Jan 2015 16:50:18 +0200
Subject: [PATCH 36/63] Fixed build flags.

---
 libvmod-querystring-4.0/src/Makefile.am | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/libvmod-querystring-4.0/src/Makefile.am b/libvmod-querystring-4.0/src/Makefile.am
index 510f32c..8f3f728 100644
--- a/libvmod-querystring-4.0/src/Makefile.am
+++ b/libvmod-querystring-4.0/src/Makefile.am
@@ -28,7 +28,7 @@
 # ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED
 # OF THE POSSIBILITY OF SUCH DAMAGE.
 
-AM_CPPFLAGS = @VMOD_INCLUDES@
+AM_CPPFLAGS = @VMOD_INCLUDES@ @DEFINE_MAJOR@ @DEFINE_LOGGING@
 
 vmoddir = @VMOD_DIR@
 vmod_LTLIBRARIES = libvmod_querystring.la
@@ -41,7 +41,7 @@ libvmod_querystring_la_SOURCES = \
 	vmod_querystring.c \
 	vmod_querystring.h
 
-vcc_if.c vcc_if.h vmod_querystring.man.rst: @VMODTOOL@ $(top_srcdir)/src/vmod_querystring.vcc ../README.rst
+vcc_if.c vcc_if.h vmod_querystring.man.rst: @VMODTOOL@ $(top_srcdir)/src/vmod_querystring@VARNISH_MAJOR@.vcc ../README.rst
 	cp ../README.rst vmod_querystring.man.rst
 	@VMODTOOL@ $(top_srcdir)/src/vmod_querystring@VARNISH_MAJOR@.vcc
 
-- 
2.7.4

